cordova-lib/src/cordova/plugin.js

```diff
@@ -185,8 +185,9 @@ module.exports = function plugin(command, targets, opts) {
                             });
                         if (missingVariables.length) {
                             shell.rm('-rf', dir);
-                            events.emit('results', 'Variable(s) missing (use: --variable ' + missingVariables.join('=value --variable ') + '=value).');
-                            return;
+                            var msg = 'Variable(s) missing (use: --variable ' + missingVariables.join('=value --variable ') + '=value).';
+                            events.emit('results', msg);
```

Comment:
There is no need to emit the results event to log error message to console. There is a an error handler in cordova-cli which does this already.